pkg/controller/siddhiprocess/artifacts.go

```diff
@@ -132,9 +132,8 @@ func (rsp *ReconcileSiddhiProcess) CreateIngress(
 			Name:      configs.HostName,
 			Namespace: sp.Namespace,
 			Annotations: map[string]string{
-				"kubernetes.io/ingress.class":                 "nginx",
-				"nginx.ingress.kubernetes.io/rewrite-target":  "/",
-				"nginx.ingress.kubernetes.io/ssl-passthrough": "true",
```

Comment:
Don't we need **ssl-passthrough** for https during default tls case

Reply:
Actually, we don't need that SSL passthrough enable. We are enabling SSL passthrough means we allow ingress to send HTTPS requests to the backend without decrypting the request. Please refer [this](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#ssl-passthrough).

Reply:
From adding this there is nothing happens to the curl -k option. We can use it as we used previously to ignore HTTPS validation.